crypto-common/src/lib.rs

```diff
@@ -163,16 +163,16 @@ pub trait KeyInit: KeySizeUser + Sized {
     #[cfg(feature = "getrandom")]
     #[inline]
     fn generate_key() -> Key<Self> {
-        Self::generate_key_with_rng(&mut OsRng)
+        Self::generate_key_with_rng(&mut OsRng).expect("RNG failure")
```

Comment:
Could potentially use a constant for "RNG failure" to ensure it's consistent everywhere, but otherwise I think this is fine